### PR TITLE
docs: new decision for adding a monitoring tool

### PR DESCRIPTION
**Pull request by gustavovalverde**
This is our first Architectural Decision Record on this project.

### CODE REVIEW

File: docs/decisions/2024-07-12-monitoring-tool.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+# status and date are the only required elements. Feel free to remove the rest.
+status: approved
+date: 2024-07-12
+story: [ZF Engineering update: 2024 Sprint 13 (June 18th - July 1st.)](https://forum.zcashcommunity.com/t/zf-engineering-update-2024-sprint-13-june-18th-july-1st/48210#:~:text=DNSSeeder%20not%20responding%20to%20requests)
+---
+
+# {short title of solved problem and solution}
```
Comment (mpguerra) [inline docs/decisions/2024-07-12-monitoring-tool.md]: Missing title here? :)

File: docs/decisions/2024-07-12-monitoring-tool.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Context & Problem Statement
+
+The Foundation has multiple production services supporting the network, and we need to monitor them, as we've recently had issues with our DNS Seeder service, and the team realizing after CI failed in Zebra. We need to have a monitoring tool to keep track of the services' health and performance, and be notified proactively when something goes wrong.
```
Comment (mpguerra) [inline docs/decisions/2024-07-12-monitoring-tool.md:12]:
```suggestion
The Foundation runs multiple production services supporting the network, and we need to monitor them, as we've recently had issues with our DNS Seeder service, that have only been noticed once CI failed in Zebra. We need to have a monitoring tool to keep track of the services' health and performance, and be notified proactively when something goes wrong.
```

**Review by mpguerra — APPROVED** [review]
Looks good, suggestions are optional

File: docs/decisions/2024-07-12-monitoring-tool.md (hunk 1 above)
Comment (gustavovalverde) [reply docs/decisions/2024-07-12-monitoring-tool.md]: It's odd GitHub showed you an old version, considering it was fixed a preview commit: [docs(adr): fix title](https://github.com/ZcashFoundation/infra-monitoring-kuma/pull/2/commits/c660da377f2ec9256a461b4825adcf844e9406e7)

**Comment by gustavovalverde** [comment]
Merging with the actual review. I might moved this (and other) decisions to https://github.com/ZcashFoundation/infra/ but in the meanwhile lets keep this one.

**Pull request merged**